src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -258,27 +335,24 @@ export class AllRecords extends React.Component<IAllRecordsProps, IAllRecordsSta
             </div>
           </div>
           <div className="sort-container">
-            {isMapView ? (
-              <div className="button-pill" onClick={this.toggleMapView}>
-                <span>
-                  <FontAwesomeIcon icon="th" />
-                  <MediaQuery minDeviceWidth={768}>
-                    &nbsp;
-                    {translate('providerSite.gridView')}
-                  </MediaQuery>
-                </span>
-              </div>
-            ) : (
-              <div className="button-pill" onClick={this.toggleMapView}>
+            <div className="button-pill" onClick={this.toggleMapView}>
```

Comment:
If we are using the ButtonPill component, then I think this should be it too. For consistency.

Reply:
I think this should be refactored in whole service provider module. Maybe as a part of #1065

Reply:
If you insist, then ok